typescript/packages/plugins/crossmint-headless-checkout/src/parameters.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from "zod";
+
+export function getCreateAndPayOrderParameters(callDataSchema: z.ZodSchema) {
```

Comment:
i think it may help to add a line of docs to each option w cursor here

Reply:
especially the locators

Reply:
sorry, i meant usign zod.describe()... i believe the llm uses that to understand what to set each param to